core/common/src/main/java/alluxio/util/network/NetworkAddressUtils.java

```diff
@@ -525,6 +525,19 @@ public static String resolveHostName(String hostname) throws UnknownHostExceptio
     return InetAddress.getByName(hostname).getCanonicalHostName();
   }
 
+  /**
+   * Resolves a given hostname IP address.
+   *
+   * @param hostname the input hostname, which could be an alias
+   * @return the hostname IP address, or null if it is null or empty
+   * @throws UnknownHostException if the given hostname cannot be resolved
+   */
+  public static String resolveIpAddress(String hostname) throws UnknownHostException {
+    Preconditions.checkNotNull(hostname);
+    Preconditions.checkArgument(!hostname.isEmpty());
```

Comment:
similar here, it's nice to put something like "Cannot resolve IP address for empty hostname"

Reply:
Done